support-frontend/assets/components/thankYou/signedIn/signedInItems.tsx

```diff
@@ -0,0 +1,107 @@
+import { css } from '@emotion/react';
+import { space } from '@guardian/source/foundations';
+import {
+	LinkButton,
+	SvgArrowRightStraight,
+} from '@guardian/source/react-components';
+import { useEffect, useState } from 'react';
+import BulletPointedList from 'components/thankYou/utilityComponents/BulletPointedList';
+import type { CsrfState } from 'helpers/redux/checkout/csrf/state';
+import { routes } from 'helpers/urls/routes';
+import { isCodeOrProd } from 'helpers/urls/url';
+import { catchPromiseHandler } from 'helpers/utilities/promise';
+
+const expandableContainer = css`
+	margin-top: ${space[4]}px;
+
+	& > * + * {
```

Comment:
Could we leave a note explaining what this is doing?

Reply:
Yes, comment added above selectors.